geode-core/src/test/java/org/apache/geode/internal/cache/LatestLastAccessTimeMessageTest.java

```diff
@@ -77,6 +78,19 @@ public void processWithNullCacheRepliesZero() {
     verify(lastAccessTimeMessage).sendReply(dm, 0);
   }
 
+  @Test
+  public void replyIsSentEvenIfThereIsAnException() {
+    setupMessage();
+    when(dm.getCache()).thenThrow(new RuntimeException());
+    try {
+      lastAccessTimeMessage.process(dm);
+      fail();
+    } catch (RuntimeException ex) {
+      // Expected
+    }
```

Comment:
Rather than using `fail()` and catching the exception just to ignore it, it might be better to extract the `new RuntimeException()` to a variable and then use `assertThatThrownBy()` to confirm that that exception is thrown.